modules/osd-create-cluster-red-hat-account.adoc

```diff
@@ -105,6 +105,11 @@ If the cluster privacy is set to *Private*, you cannot access your cluster until
 You can review the end-of-life dates in the update life cycle documentation for {product-title}. For more information, see _{product-title} update life cycle_.
 ====
 +
+.. Provide administrator approval per your cluster update method:
+** Individual updates: If you select an update version that requires approval, provide an administrator’s acknowledgment and click *Approve and continue*.
+** Recurring updates: If you selected recurring updates for your cluster, provide an administrator’s acknowledgment and click *Approve and continue*. The {cluster-manager} does not start scheduled updates without receiving an administrator's acknowledgment.
```

Comment:
y-stream upgrade is controlled by another attribute enable_minor_version_upgrades set to true which is not supported on UI yet.